fixture-monkey-junit-jupiter/src/main/java/com/navercorp/fixturemonkey/junit/jupiter/annotation/ShowMeLog.java

```diff
@@ -0,0 +1,12 @@
+package com.navercorp.fixturemonkey.junit.jupiter.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD})
+public @interface ShowMeLog {
+	String value() default "";
```

Comment:
When it returns the `String`, it should check if the `value()` is of long type.

So how about changing it to long type?

Reply:
I understand Thasnk for your Feeadback